_episodes/03-configuration.md

```diff
@@ -3,91 +3,178 @@ title: "Configuration"
-  * Directory structure for input data
-  * Number of available CPUs
-  * Destination directory
-  * Auxiliary data directory
-  * Output settings
-
-## Rootpaths to input data
-ESMValTool uses several categories (in ESMValTool, this is referred to as projects) for input data based on their source, like CMIP for dataset from climate model intercomparison project, and OBS for observational dataset that adhere to (CMOR standard)[https://cmor.llnl.gov/].
-For each category, you can define either one path or several pathes as a list.
-In this lesson, you work with data from (CMIP5)[https://esgf-node.llnl.gov/projects/cmip5/].
-Add the root path of the folder where you downloaded the data during the (Setup)[https://escience-academy.github.io/lesson-esmvaltool/setup.html].
 
+* Rootpath to input data
+* Directory structure for the data from different projects
+* Number of parallel tasks
+* Destination directory
+* Auxiliary data directory
+* Output settings
+
+> ## Which text editor
+>
+> No matter what editor you use, you will need to know where it searches for and saves files. If you start it from the shell, it will (probably) use your current working directory as its default location. We use ``nano`` in examples because it is one of the least complex text editors. Press <kbd>ctrl</kbd> + <kbd>O</kbd> to save the file, and then <kbd>ctrl</kbd> + <kbd>X</kbd> to exit ``nano``.
+{: .callout}
+
+## Rootpath to input data
+
+ESMValTool uses several categories (in ESMValTool, this is referred to as projects) for input data based on their source.
+The current categories in the configuration file are mentioned below. For example, CMIP is used for a dataset from the climate model intercomparison project whereas OBS for an observational dataset. The ``rootpath`` specifies the directories where ESMValTool will look for input data. For each category, you can define either one path or several paths as a list.
+
+~~~YAML
+rootpath:
+  CMIP3: [~/cmip3_inputpath1, ~/cmip3_inputpath2]
+  CMIP5: [~/cmip5_inputpath1, ~/cmip5_inputpath2]
+  CMIP6: [~/cmip6_inputpath1, ~/cmip6_inputpath2]
+  OBS: ~/obs_inputpath
+  OBS6: ~/obs6_inputpath
+  obs4mips: ~/obs4mips_inputpath
+  ana4mips: ~/ana4mips_inputpath
+  native6:  ~/native6_inputpath
+  RAWOBS: ~/rawobs_inputpath
+  default: ~/default_inputpath
 ~~~
+
+In this lesson, we will work with data from [CMIP5](https://esgf-node.llnl.gov/projects/cmip5/).
+We add the root path of the folder where data is available.
+
+~~~YAML
   rootpath:
   ...
-    CMIP5: [~/cmip5_inputpath1, ~/cmip5_inputpath2, ~/escience-academy/test_data]
+    CMIP5: [~/cmip5_inputpath1, ~/cmip5_inputpath2, ~/esmvaltool_tutorial/data]
 ~~~
-{: .source}
 
-## Auxiliary data directory (used for some additional datasets)
-  auxiliary_data_dir: ~/auxiliary_data
+> ## Setting the correct rootpath
+>
+> * To get the data (or its correct rootpath), check instruction in [Setup](https://esmvalgroup.github.io/tutorial/setup.html).
```

Comment:
I am not sure what you mean here. Is this a pointer toward instructions on where to actually get the CMIP5 data or is this a pointer toward the correct setup of the rootpath?

Reply:
The instruction on how to get the data (or its rootpath) will come in setup section of the tutorial. This is the most common approach for several lessons that I have seen in software carpentry. The setup section contains everything related to data and additional material. Therefore, the episode content won't be changed. Please also see the discussion in the comment [here](https://github.com/ESMValGroup/tutorial/pull/12#discussion_r393092548)

Reply:
Ok.